requirements/pytorch/test.txt

```diff
@@ -1,18 +1,16 @@
-coverage>=6.4
-codecov>=2.1
-pytest>=7.0
-pytest-cov
-pytest-forked
+coverage>=6.4, <=6.4.2
+codecov>=2.1, <=2.1.12
+pytest>=7.0, <=7.1.2
+pytest-cov <=3.0.0
+pytest-forked <=1.4.0
 pytest-rerunfailures>=10.2
-mypy>=0.920
```

Comment:
Why remove `mypy` from here? Having it here means dependabot will try to upgrade it. Also, it's useful for contributors who just install all test requirements.

Reply:
Yes, `mypy` is used in a specific case and for such you do not need to install all testing requirements such as pytest with all plugins...

Reply:
If mypy and flake8 are part of pre-commit and pre-commit is part of a CI job, I consider them testing requirements.

You don't want contributors to have to look inside the CI jobs to know what to install. IMO this fits here.

Additionally, there's the benefit of getting dependabot upgrades.

Reply:
yes, `flake8` is part of pre-commit, but Mypy not as if you run all files, it takes too much time, but in the normal scenario, it runs only on changed files, so shall we consider adding back `mypy` as a pre-commit hook?

Reply:
Sorry, my mistake.

`mypy` is not part of pre-commit, but still part of CI, so the testing requirements should also install it in my view